glossary.md

```diff
@@ -0,0 +1,33 @@
+[Back to Conversation AI Overview](https://conversationai.github.io/) | [Back to Perspective API documentation](https://github.com/conversationai/perspectiveapi/blob/master/README.md)
+
+# Glossary
+
+These are the terms you must know to understand the Perspective API and documentation.
+
+### Attribute
+
+A comment label used for review. The most popular attributes are the toxicity and severe toxicity models.
+
+### Comment
+
+The text to be scored. Each API request contains a single comment. A comment could be a single post to a web page's comments section, a forum post, a message to a mailing list, a chat message, etc.
+
+### Model
+
+A dimension that the comment is scored on, for example, "toxic", "obscene", "thoughtful", "off-topic", etc. This is also sometimes referred to as an attribute or model attribute. Models can be labeled Alpha, meaning they are recommended for use, or experimental, meaning that they are still in development but are open for public use if you're comfortable using experimental, imperfect models. See our list of models here.
```

Comment:
change to "Models can be labeled Prod" because that's how they're labeled in the API reference table?